src/importchecker/importchecker.py

```diff
@@ -18,7 +18,7 @@
 
 $Id$
```

Comment:
My fingers are itching to mass-remove these Subversion crumbs.

Reply:
fixed